packages/liferay-npm-scripts/package.json

```diff
@@ -39,13 +39,14 @@
 		"deepmerge": "^4.0.0",
 		"eslint": "6.5.1",
 		"eslint-config-liferay": "13.0.0",
+		"fs-extra": "^8.1.0",
```

Comment:
🤦‍♂️ we now have four copies of this dependency in this project alone:

```
$ yarn why fs-extra
yarn why v1.17.3
[1/4] 🤔  Why do we have the module "fs-extra"...?
=> Found "fs-extra@8.1.0"
info Has been hoisted to "fs-extra"
info Reasons this module exists
   - "workspace-aggregator-7e6f7756-ea7d-478a-b6cb-391cbafc5d06" depends on it
   - Hoisted from "_project_#liferay-npm-scripts#fs-extra"
   - Hoisted from "_project_#liferay-npm-scripts#liferay-npm-bridge-generator#fs-extra"
   - Hoisted from "_project_#liferay-npm-bundler-preset-liferay-dev#liferay-npm-bundler-plugin-replace-browser-modules#fs-extra"
   - Hoisted from "_project_#liferay-npm-scripts#liferay-npm-bundler#fs-extra"
=> Found "liferay-theme-tasks#fs-extra@0.24.0"
info This module exists because "_project_#liferay-npm-scripts#liferay-theme-tasks" depends on it.
=> Found "@storybook/core#fs-extra@8.0.1"
info This module exists because "_project_#liferay-npm-scripts#@storybook#react#@storybook#core" depends on it.
=> Found "file-system-cache#fs-extra@0.30.0"
info This module exists because "_project_#liferay-npm-scripts#@storybook#react#@storybook#core#file-system-cache" depends on it.
```

liferay-portal (without this PR) has 6:

```
=> Found "fs-extra@0.24.0"
=> Found "liferay-npm-bundler#fs-extra@4.0.3"
=> Found "liferay-npm-bridge-generator#fs-extra@5.0.0"
=> Found "liferay-npm-bundler-plugin-replace-browser-modules#fs-extra@7.0.1"
=> Found "@storybook/core#fs-extra@8.1.0"
=> Found "file-system-cache#fs-extra@0.30.0"
```

Reply:
What do you suggest? That's always going to happen as there are projects we don't have control over.

Maybe we can use the same version for our projects but again it's going to be difficult to make them converge always. It can even lead to needing spurious releases just to make sure all our projects share the same version...

However, it's strange that `liferay-npm-bundler`, `liferay-npm-bridge-generator` and `liferay-npm-bundler-plugin-replace-browser-modules` use different versions since I unified that 1-2 months ago. It is a smell. Maybe the portal is using more than one version of the JS Toolkit too and that's the root cause.

Reply:
> What do you suggest? That's always going to happen as there are projects we don't have control over.

Nothing sadly, short of avoiding using dependencies when it is practical to do so. I don't even know what you're using from `fs-extra` here; is it just `ensureDirSync` or something else as well?

Reply:
I'm using `removeDirSync` instead of  `rimraf` to get rid of the whole directory tree. We may as well leave the folder tree empty, but I preferred to delete everything to make sure leftovers don't interact with the build.

Reply:
> Nothing sadly, 

:cry: